centml/compiler/config.py

```diff
@@ -31,5 +36,10 @@ class Config(BaseSettings):
     # If the server response is smaller than this, don't gzip it
     MINIMUM_GZIP_SIZE: int = 1000
 
+    MODE: OperationMode = OperationMode.PREDICTION
+    PREDICTION_DATA_DIR: str = os.path.join(os.path.dirname(__file__), 'sample_data.csv')
```

Comment:
lets have this variable directly point to the file instead of directory. Gives more flexibility on which filename to use.